controllers/auth.js

```diff
@@ -135,5 +135,21 @@ const updatepassword = async (req, res) => {
     return errorResponse(res, 500, "server error");
   }
 };
+//api  for getting the user detail for admin
+const getusers=async(req,res)=>{
```

Comment:
format the function for a cleaner code base (remove all unused code, have a proper indentation, use logs where ever required)

Recommendation user **Prettier**